Clappr/Classes/Base/UIObject.swift

```diff
@@ -0,0 +1,5 @@
+protocol UIObject {
+    var view: UIView? { get set }
```

Comment:
I think that is better to just provide get in order to set the concrete implementation as private(set), what you think?

Reply:
I agree! :)

I will change it, tks. 👍